step-ca/configuration.mdx

```diff
@@ -682,6 +685,28 @@ When you start `step-ca`, it prints your X.509 root fingerprint, and the SSH use
 2022/09/20 16:28:45 Serving HTTPS on :443 ...
 ```
 
+#### Notes on Google Cloud KMS Permissions
```

Comment:
The permissions here are only for `step-ca`. They won't work for creating keys using `step-kms-plugin`. The same happens for Azure and AWS.